packages/instrumentor/plugins/functionHooks.test.ts

```diff
@@ -1012,18 +1012,31 @@ function expectLogHooks(
 	}
 }
 
-function expectTrackedHooks(tracker: Set<string>, entries: string[]) {
-	expect(entries.every((e) => tracker.has(e))).toBeTruthy();
-	expect(tracker.size).toEqual(entries.length);
+// Only given functionNames (and none else) should be present in trackedHooks.
+function expectTrackedHooks(
+	trackedHooks: TrackedHook[],
+	functionNames: string[]
+) {
+	for (const functionName of functionNames) {
+		let found = false;
+		for (const hook of trackedHooks) {
+			if (hook.target === functionName) {
+				found = true;
+				break;
+			}
+		}
+		expect(found).toBeTruthy();
+	}
```

Comment:
Can we get rid of the nested for loops by turning the old check around? Something like `expect(trackedHooks.every(h => functionNames.includes(h.target))).toBeTruthy();`